zerver/lib/upload.py

```diff
@@ -190,10 +207,22 @@ def upload_avatar_image_local(user_file, user_profile, email):
 if settings.LOCAL_UPLOADS_DIR is not None:
     upload_message_image = upload_message_image_local
     upload_avatar_image  = upload_avatar_image_local
+
 else:
     upload_message_image = upload_message_image_s3
     upload_avatar_image  = upload_avatar_image_s3
 
+
+def claim_attachment(path_id, message):
+    #update attachment entry in DB
+    try:
+        Attachment.objects.filter(path_id=path_id).update(message=message)
+        return True
+    except Attachment.DoesNotExist:
+        raise JsonableError("The upload was not successful. Please reupload the file again in a new message.")
```

Comment:
Oh no. I sort of messed up my local repo and had to rewrite some of the code again. The exception did not pop up when I tested it last time. I did change the code but I lost that commit.